articles/tutorials/create-a-sample-app/02-create-a-petclinic-csharp/create-a-petclinic-csharp.md

```diff
@@ -220,43 +255,43 @@ Once the staged `Changes` comes into view, you can review the proposed code chan
 
 Click on the file located in the `Implementation` folder.
 
-![Owner Rest Controller](images/software-factory-run-staging-ownerservice.png)
+![View of the Software Factory Changes](images/software-factory-run-staging-ownerservice.png)
 
-On clicking the file, Intent Architect will start up an external "diff" tool which you can use to review the changes. The diff tool used by Intent Architect can be configured manually or alternatively Intent Architect will try see if Visual Studio Code is already installed and use it.
+On clicking the file, Intent Architect will start up an external "diff" tool which you can use to review the changes. By default Intent Architect will use Visual Studio Code if it can find it, otherwise you can manually configure any other "diff" tool in Settings.
 
-![Service Implementation](images/software-factory-run-staging-diff-ownerrestcontroller-impl.png)
+![A "diff" showing the staged contents of OwnerService.cs](images/software-factory-run-staging-diff-ownerrestcontroller-impl.png)
 
-For the purposes of this tutorial, don't apply the changes at this time and instead click `CANCEL`.
+Don't apply the changes for the moment and instead click `CANCEL`.
 
 Click the `Modules` option located in the left pane of the window and install the following modules:
 
 - `Intent.EntityFrameworkCore.Repositories`
 - `Intent.Application.ServiceImplementations.Conventions.CRUD`
 
-Execute the Software Factory again, click the file `OwnerService` in the changes and observe how its output has changed from the previous run..
+Execute the Software Factory again, click the file `OwnerService` in the changes and observe how its output has changed from the previous run.
 
 ![Service Implementation CRUD](images/software-factory-run-staging-diff-ownerrestcontroller-impl-crud.png)
 
 > [!NOTE]
-> Observe that the operations are now populated with code that will facilitate CRUD operations.
+> Observe that the operations are now populated with code that will facilitate [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations.
 >
 > The `Intent.Application.ServiceImplementations.Conventions.CRUD` module automatically checked for operations which followed simple conventions and generated basic implementations where found.
 >
 > Anything not meeting these conventions was ignored and no implementations were generated.
-
+>
 > [!WARNING]
 >
-> It is strongly recommended that one _always_ reviews all files in the `Changes` tab of the Software Factory Execution to ensure that all changes that will be applied are what you expect.
+> It is recommended to always review all files in the `Changes` tab of the Software Factory Execution to ensure that all changes that will be applied are as expected.
```

Comment:
I deliberately de-emphasized this since as people develop a trust in Intent Architect, for some changes they will skip reviewing them, or be content to use their SCM to double check. So this is just "advice" now rather than a strong recommendation.